docs/user_guides/fs/feature_view/model-dependent-transformations.md

```diff
@@ -0,0 +1,131 @@
+# Model Dependent Transformation Functions
+
+
+[Model-dependent transformations](https://www.hopsworks.ai/dictionary/model-dependent-transformations) generate feature data tailored to specific models and often require the computation of training dataset statistics. Hopsworks enhances the robustness of AI pipelines by preventing [training-inference skew](https://www.hopsworks.ai/dictionary/training-inference-skew) by ensuring that the same model-dependent transformations and statistical parameters are used during both training dataset generation and online inference.
```

Comment:
transform feature data for a specific model. Feature encoding is one example of such a transformations. Feature encoding is parameterized by statistics from the training dataset, and, as such, many model-dependent transformations require the training dataset statistics as a parameter.